code_map.py

```diff
@@ -136,8 +147,8 @@ def get_group(view):
         if w:
             return w.get_view_index(view)[0]
 
-    # I wish C# style checking was possible with Python: 
-    # return view?.window()?.get_view_index(view)[0]          
+    # I wish C# style checking was possible with Python:
+    # return view?.window()?.get_view_index(view)[0]
 
```

Comment:
Please set:

`"trim_trailing_white_space_on_save": false,`

in `Preferences -> Settings` or at least in `Preferences -> Settings - Syntax specific`

PLEASE.. every time the diff looks like this. Python doesn't want trailing white spaces.